introduction.md

```diff
@@ -1,3 +1,6 @@
 # I am the master of this dojo!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?